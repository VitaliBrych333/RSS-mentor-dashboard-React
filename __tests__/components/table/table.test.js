import React from 'react';
import { shallow, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import Table from '../../../src/components/table';

configure({ adapter: new Adapter() });

describe('Table ', () => {
    test('match snapshot', () => {
        const wrapper = shallow(<Table />);
        expect(wrapper).toMatchSnapshot();
    });
});
