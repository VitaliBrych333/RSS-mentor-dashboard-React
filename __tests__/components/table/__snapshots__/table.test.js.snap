// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table  match snapshot 1`] = `""`;
